src/app/src/utils/index.js

```diff
@@ -18,6 +18,8 @@ export const isQuestionCode = key => {
     return key.includes('.');
 };
 
+const isCellEmpty = value => value == null || value.length === 0;
```

Comment:
Consider empty strings `null` values; by explicitly comparing to 0 instead of using `!value.length`, this avoids marking valid values without a `length` property null